cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/template/etl/realtime/source/TwitterSource.java

```diff
@@ -48,7 +48,9 @@
  */
 @Plugin(type = "source")
 @Name("Twitter")
-@Description("Twitter Realtime Source")
+@Description("Twitter Realtime Source. Output records will always contain id (long), message (string), " +
+  "favCount (int), rtCount (int), and isRetweet (boolean) fields. Output records may contain lang (string), " +
+  "time (long), source (string), geolat (double), and geoLong (double) fields if they are available.")
```

Comment:
Not part of this PR but can we rename the GLAT field from geolat to geoLat ? to be in sync with geoLong.